### PR TITLE
docs: crontab & job scheduler docs

### PR DESCRIPTION
**Pull request by stlgaits**

### CODE REVIEW

File: docs/cron_jobs.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+| Command          | your command's name                                                                                                                                                                                                                                                                                                                                                               |
+| Period           | just like the job scheduler, period is a time interval corresponding to the when your task will be executed. <br/> Here on the other hand the syntax to use is very precise. <br/> Example : */5* ** * <br/> in order of writing:  minute(0-59), hour (0-23), day of the month(1-31), month(1-12), day of the week(0-6) (Sunday to Saturday => 7 is also Sunday on some systems). |
+| Arguments        |                                                                                                                                                                                                                                                                                                                                                                                   |
```
Comment (stlgaits) [inline docs/cron_jobs.md:89]: @CamillePMyddleware @Myddleware I need help for these sections of the table as I'm not too familiar with crontab

**Comment by stlgaits** [comment]
@Myddleware We can merge these changes but the following sections of the documentation are still missing : 
- Upgrading Myddleware (inside Admin tasks)
- Myddleware API
- Advanced Usage (Filters, Relationships, mass actions) going deeper etc)
- Developer's guide needs to be thoroughly reviewed + some images are missing

**Comment by stlgaits** [comment]
The Upgrade Myddleware section will require some review @Myddleware, especially regarding the custom code section which I wasn't too sure what to do with

**Comment by stlgaits** [comment]
@Myddleware Could you merge this branch as it is for now? 
Some parts are still missing but we can add all these new additions already to improve the current version 😃

**Comment by Myddleware** [comment]
Hi @EstelleMyddleware.
Done
Thanks.

**Pull request merged**